### ISSUE TITLE
FF: Arrow keys in Files view interpreted as Delete

### ISSUE THREAD

**Issue by jhpedemonte** [maqetta/maqetta#3887]
Select a file in the Files view and attempt to use the arrow keys.  The action is interpreted as **delete** -- a dialog asking for your confirmation to delete the file appears.

**Comment by peller** [comment]
somehow the delete button and others on the files view are left out of the tab order. may be related to the new project buttons. @JonFerraiolo

**Comment by JonFerraiolo** [comment]
I'm using `float:right` for the 4 buttons on the right. Could that contribute to the problem?

**Comment by peller** [comment]
nah, my mistake, toolbar items aren't in the tab order anyway. you have to arrow key through them (and that's working, just backwards for the `float:right` elements, which ought to be ok)

**Comment by peller** [comment]
the problem seems to be specific to FF
